cms/static/js/views/utils/xblock_utils.js

```diff
@@ -86,12 +86,14 @@ define(["jquery", "underscore", "gettext", "common/js/components/utils/view_util
                         }
                     );
                 },
-                messageBody = interpolate(
+                messageBody;
+            xblockType = xblockType || 'component';
```

Comment:
And here it is fixed on the sandbox--

![image](https://cloud.githubusercontent.com/assets/484484/15636169/87425c44-25c4-11e6-9576-9fd486137d09.png)